app/views/handlers/list_action.py

```diff
@@ -32,14 +34,21 @@ def is_location_valid(self):
         return True
 
     def get_previous_location_url(self):
-        block_id = self._request_args.get("previous") or self._request_args.get(
-            "return_to"
-        )
+        if self._request_args.get("return_to_summary"):
```

Comment:
`self._request_args.get("return_to_summary")` is used 7 times, can we add a top level property to block handler so that it is defined in one place?
```py
self._return_to_summary = self._request_args.get("return_to_summary")
```